### ISSUE TITLE
Provide some general information about the split in the t.protocols' docstring

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2027]
|[<img alt="jml's avatar" src="https://avatars.githubusercontent.com/u/125674?s=50" width="50" height="50">](https://github.com/jml)| @jml reported|
|-|-|
|Trac ID|trac#2027|
|Type|enhancement|
|Created|2006-08-29 07:04:32Z|
|Branch|https://github.com/twisted/twisted/tree/protocol-split-doc-2027|

The protocols docstring should probably explain what went on with the split. e.g. It should explain that many of the protocols there can be found in their respective packages.

Don't use that sentence though. That's a terrible sentence.

<details><summary>Searchable metadata</summary>

```
trac-id__2027 2027
type__enhancement enhancement
reporter__jml jml
priority__low low
milestone__ 
branch__branches_protocol_split_doc_2027 branches/protocol-split-doc-2027
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1156835072000000 1156835072000000
changetime__1220890822000000 1220890822000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)|@glyph set owner to @radix|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_2"></a>|@thijstriemstra commented|
|-|-|

(In [24683]) Branching to 'protocol-split-doc-2027'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra removed owner|
|-|-|

See r24684, up for review.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun set owner to @thijstriemstra|
|-|-|

Okay, great, please merge.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [24724]) Merge protocol-split-doc-2027: Provide some general information about the split in the twisted.protocols docstring.

Author: thijs
Reviewer: exarkun
Fixes: [#2027](https://github.com/twisted/twisted/issues/2027)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
